### PR TITLE
Add next platform cicd

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Adds ci/cd for the new platform

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Follow the [these setup instructions](https://github.com/airqo-platform/AirQo-frontend/blob/ft-nextPlatform-cicd/platform/README.md#setup) in the [README](https://github.com/airqo-platform/AirQo-frontend/blob/ft-nextPlatform-cicd/platform/README.md)
* Build the container: `docker build -t next-platform .`
* Run the container: `docker run -p 3000:3000 next-platform`
* Access [localhost:3000](http://localhost:3000/)

#### What are the relevant tickets?
- [OPS-40](https://airqoteam.atlassian.net/browse/OPS-40?atlOrigin=eyJpIjoiNWU0YWM0ODQxNjI5NDc1ODhlZWQ2MDc5NTIxYTQ2MzgiLCJwIjoiaiJ9)

#### Screenshots (optional)
image

### CODE REVIEW

**Review by Baalmart — CHANGES_REQUESTED** [review]
Thanks for the great works @123MwanjeMike 
Please create a new PR. Your PR base branch is the same as the source of the changes.

**Comment by 123MwanjeMike** [comment]
> Thanks for the great works @123MwanjeMike
> Please create a new PR. Your PR base branch is the same as the source of the changes.

This PR is a build up [to this PR](https://github.com/airqo-platform/AirQo-frontend/pull/759). The base is intentionally not staging.

File: k8s/platform/prod-airqo-next-platform.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      app: airqo-next-platform
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: airqo-next-platform
+    spec:
+      containers:
+        - name: airqo-next-platform
+          image: us.gcr.io/airqo-250220/airqo-next-platform:latest
```
[the remark this answers is not in the document]
Comment (123MwanjeMike) [reply k8s/platform/prod-airqo-next-platform.yaml:21]: Hi @NoahNsimbe 
The image is built from the production deploy workflow. This is the file
https://github.com/airqo-platform/AirQo-frontend/blob/bf94e359951dcbcd4eb16b9fc8118f69467d7f01/.github/workflows/deploy-frontends-to-production.yml#L312

[the remark this answers is not in the document]
Comment (NoahNsimbe) [reply k8s/platform/prod-airqo-next-platform.yaml:21]: great

**Comment by Baalmart** [comment]
> > Thanks for the great works @123MwanjeMike
> > Please create a new PR. Your PR base branch is the same as the source of the changes.
> 
> This PR is a build up [to this other PR](https://github.com/airqo-platform/AirQo-frontend/pull/759). The base is intentionally not staging. cc @Baalmart

Alright @123MwanjeMike , I am still curious about this Git workflow. How does our Github policy relate with this workflow taken here?

As always, I usually discourage any "unique awesome ways" of doing things. These unique and awesome Git workflows usually bring avoidable problems in the near future.

**Pull request merged**